episodes/chapter3.md

```diff
@@ -0,0 +1,120 @@
+- For the other collaboration, which organization you would need to reach in order to fix the issue?
+
+You will now link the 'community-store' to the collaboration whose nodes are ready for it. 
+
+1. Login into the organization using the corresponding credentials from above. 
+2. Click on 'Administration' on the top of the UI, select 'Collaborations' on the left panel, and then select the corresponding collaboration.
+3. Click on '+ Add algorithm store'
+4. Add the vantage6's community store. Use any descrption as name, and provide community store URL: `https://store.cotopaxi.vantage6.ai`
+5. Make sure the store is now shown on the collaboration details:
+![Community store entry on the collaboration details](fig/chapter3/community-store-entry.png)
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+## Challenge 3: your first algorithm execution as a researcher
+
+Now, you'll take on the role of the researcher within the collaboration for which you've just established the algorithm store. With this role, you will finally request the execution of the algorithm. 
+
+1. log in as a researcher using the corresponding credentials below:
+
+    |  User  |  Roles   |  Collaboration   |
+    |----|-----|-----|
+    |PhY24-rs1  | Researcher    |PhY24      |
+    |GHT-rs1  | Researcher    |GHT    |
+
+2. Select the collaboration given on the front page, and select 'Tasks' from the panel on the left.
+    ![Collaboration researcher view](fig/chapter3/collab-researcher-view.png)
+
+3. If you have set up everything correctly, the 'Average' algorithm should be now listed under the '*Select which algorithm you want to run*' dropdown menu. Select it, and provide a name and a description.
+
+    ![Algorithm selection](fig/chapter3/task-alg-selection.png)
+
+4. Now the UI will let you choose between the two functions you explored in Challenge #1. First, try to run the 'partial_average' on all the nodes individually.
+
+    ![Running a function on all nodes](fig/chapter3/task-partial-on-individial-orgs.png)
+
+5. Select the 'default' database, choose any numerical column as a parameter, and then click on 'Submit'.
+
+6. The task you just requested should be listed with a 'pending' status. Once finished, explore and download the provided results:
+    ![alt text](fig/chapter3/task-results.png)
+
+
+   - Based on your understanding of the 'central_average' function, if you create one a new task, which organization nodes should you choose this time in order to actually calculate the overall (across all the datasets) average? Experiment with this and discuss the results with the instructors.
+   - What would happen if you select an alpha-numerical column (e.g., 'participant_pseudo_id')? Do this experiment and explore the generated error logs. Discuss with the instructors how these logs can be used to diagnose any task execution issues.
```

Comment:
I think this is super important, and I think this should be its own Challenge. Set them up with a task to fail and let them figure out from the logs what went wrong. We might have to update the algorithm a little to make it easier for the participant.